src/sentry/api/endpoints/organization_spans_trace.py

```diff
@@ -34,46 +33,31 @@ class OrganizationSpansTraceEndpoint(OrganizationEventsV2EndpointBase):
         "GET": ApiPublishStatus.PRIVATE,
     }
 
-    @sentry_sdk.tracing.trace
-    def query_trace_data(self, snuba_params: SnubaParams, trace_id: str) -> list[SnubaTrace]:
-        builder = SpansIndexedQueryBuilder(
-            Dataset.SpansIndexed,
-            params={},
-            snuba_params=snuba_params,
-            query=f"trace:{trace_id}",
-            selected_columns=[
-                "span.op",
-                "span.description",
-                "id",
-                "is_transaction",
-                "precise.start_ts",
-                "precise.finish_ts",
-            ],
-            orderby="precise.start_ts",
-            limit=10_000,
+    def serialize_rpc_span(self, span: RPCSpan) -> SerializedEvent:
```

Comment:
This serialization probably needs to be redone completely so that the FE can actually use this data